Bottom: 62b8205caeb617d7d661dc92c0ad1b6d60e0ba19
Top:    e80d5b85ada43a0c6d87988ba0ed140395d0bfc5
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-15 01:55:26 +0000

Refresh of bga-wizard

---

diff --git a/src/Makefile.am b/src/Makefile.am
index a2d4ed7..bb3513c 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -307,15 +307,15 @@ LIBGTK_SRCS = \
 	hid/gtk/ghid-main-menu.h \
 	hid/gtk/ghid-route-style-selector.c \
 	hid/gtk/ghid-route-style-selector.h \
-	hid/gtk/gtkhid.h \
 	hid/gtk/gtkhid-main.c \
+	hid/gtk/gtkhid.h \
+	hid/gtk/gui.h \
 	hid/gtk/gui-command-window.c \
 	hid/gtk/gui-config.c \
-	hid/gtk/gui-dialog.c \
 	hid/gtk/gui-dialog-print.c \
+	hid/gtk/gui-dialog.c \
 	hid/gtk/gui-drc-window.c \
 	hid/gtk/gui-drc-window.h \
-	hid/gtk/gui.h \
 	hid/gtk/gui-keyref-window.c \
 	hid/gtk/gui-library-window.c \
 	hid/gtk/gui-library-window.h \
